Soccar [Client]/src/nl/soccar/ui/physics/models/CarPhysics.java

```diff
@@ -1,72 +1,65 @@
 package nl.soccar.ui.physics.models;
 
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.List;
 import nl.soccar.library.Car;
+import nl.soccar.ui.input.Keyboard;
 import nl.soccar.ui.physics.PhysicsContants;
 import nl.soccar.ui.physics.WorldObject;
+import nl.soccar.ui.physics.enumeration.HandbrakeAction;
 import nl.soccar.ui.physics.enumeration.SteerAction;
 import nl.soccar.ui.physics.enumeration.ThrottleAction;
 import nl.soccar.util.PhysicsUtilities;
 import org.jbox2d.collision.shapes.PolygonShape;
-import org.jbox2d.dynamics.Body;
-import org.jbox2d.dynamics.BodyDef;
-import org.jbox2d.dynamics.BodyType;
-import org.jbox2d.dynamics.FixtureDef;
-import org.jbox2d.dynamics.World;
+import org.jbox2d.dynamics.*;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
 
 /**
- * CarPhysics is a physics-model that keeps track of the physics of a Car.
- * 
+ * CarPhysics is a physics-model that keeps track of the physics of the Car.

```

Comment:
In mijn IDE was de originele comment verdwenen dus deze verander ik weer terug naar "a Car".